SPONSORSHIP.md

```
@@ -0,0 +1,109 @@
+- **US residents:** Form W-9
+- **Non-US residents:** Form W-8BEN
+
+- [Tax information for GitHub Sponsors](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/tax-information-for-github-sponsors)
+
+---
+
+### Step 6 — Submit for GitHub Review
+
+Click **Request approval** from your Sponsors dashboard. Once approved, your profile goes live at:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)